### PR TITLE
Components es6 apply code review fix suggestions

### PR DESCRIPTION
**Pull request by Swiftb0y**
These are some of the minimally invasive fixes I suggested during code review. I don't have have access to my controller atm so it'll take some time until I'll be able to implement other stuff I suggested during review.

### CODE REVIEW

File: res/controllers/midi-components-0.1.js [hunk 1]
```diff
@@ -357,6 +358,7 @@
             options.group = "[Sampler" + options.number + "]";
             options.outKey = null, // hack to get Component constructor to call connect()
             super(options);
+            // this.connect();
```
Comment (Be-ing) [inline res/controllers/midi-components-0.1.js]: Did you commit this on accident?

File: res/controllers/midi-components-0.1.js [hunk 2]
```diff
@@ -125,6 +125,7 @@
             }
         }
         trigger() {
+
```
Comment (Be-ing) [inline res/controllers/midi-components-0.1.js:128]: stray line

File: res/controllers/midi-components-0.1.js (hunk 1 above)
Comment (Swiftb0y) [reply res/controllers/midi-components-0.1.js]: Ups, yes this was on accident. It was left from a previous experiment where I tried to call connect after the parent constructor but that didn't work out-right so I wanted to take a look some other time.

File: res/controllers/midi-components-0.1.js (hunk 2 above)
Comment (Swiftb0y) [reply res/controllers/midi-components-0.1.js:128]: eslint broke when I removed it. Seems to me like some obscure bug but idk.

**Comment by Swiftb0y** [comment]
Whats left to do here?

**Comment by Be-ing** [comment]
will take another look soon. I want to get this merged for 2.4 beta

**Comment by Swiftb0y** [comment]
I still wouldn't really want to ship it for 2.4 but getting it into main ASAP would be good regardless.

**Pull request closed without merging**